composer.json

```diff
@@ -23,6 +23,7 @@
         "psr/container": "^1.0",
         "psr/log": "~1.0",
         "psr/simple-cache": "^1.0",
+        "fig/link-util": "^1.0",
```

Comment:
We use the implementations here: https://github.com/symfony/symfony/pull/22273/files#diff-78128eb201bbbb556131b3ae4bac192bR29.
We may require only a `psr/link-implementation` instead of `fig/link-util` but it will hurt the DX (the constructor argument is optional). I'm to let it as is because you can already use any implementation but it also works out of the box.